src/init/starship.bash

```diff
@@ -61,9 +61,9 @@ else
     fi
 
     # Finally, prepare the precmd function and set up the start time.
-    PROMPT_COMMAND="starship_precmd;$PROMPT_COMMAND"
+    PROMPT_COMMAND="$PROMPT_COMMAND;starship_precmd"
```

Comment:
I don't know if this change was intentional, but it appears to have broken bash initialization for me:

![image](https://user-images.githubusercontent.com/4658208/70674934-7c370580-1c55-11ea-961f-8898830c49ca.png)

Reverting this change seems to solve the problem for me.

Reply:
I did this change because the cmd_duration feature was broken on my bash configuration. I recently switch to Manjaro Linux and there the default bash configuration sets the  PROMPT_COMMAND variable and breaking the cmd_duration feature.
Changing the order of the concatenation seems to fix the problem for me.

Check my new commit. I hope to have fixed your issue.
In addition to that, I added a check to avoid the concatenation of multiple sharship_cmd function ones after the other in the case of multiple calls to `eval "$(starship init bash)"`.